experimental/Schemes/SpecOpen.jl

```diff
@@ -711,13 +711,13 @@ function Base.show(io::IO, f::SpecOpenMor)
   #given by the rational map $(generic_fractions(f))")
 end
 
-function SpecOpenMor(U::T, V::T, f::Vector{<:RingElem}; check::Bool=true) where {T<:SpecOpen}
+function SpecOpenMor(U::SpecOpen, V::SpecOpen, f::Vector{<:RingElem}; check::Bool=true)
```

Comment:
Why change this (here and elsewhere) ? Do you want to allow two `SpecOpen` of a different type?

Reply:
Yes, exactly. We need that flexibility.